patches/tModLoader/Terraria/ModLoader/Core/AssemblyManager.cs

```diff
@@ -232,7 +243,7 @@ private class LoadedMod
 			AppDomain.CurrentDomain.AssemblyResolve += (_, args) => {
 				string name = new AssemblyName(args.Name).Name;
 
-				if (name == "Terraria")
+				if (name == "Terraria" || name.Contains("tModLoader"))
```

Comment:
reason for removal?

Reply:
Terraria's assembly name for netcore was tModLoader to allow it to properly run on .NET Core—filename and assemblyname had to match. I don't quite remember if this is deprecated, but I know it doesn't hurt.

Reply:
I read this wrong, had the original and changed sides backwards. This makes sense to me. I'd prefer `StartsWith` but all good.